frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -399,14 +399,17 @@ export async function apply(body: KubeObjectInterface): Promise<JSON> {
   }
 }
 
-export async function metrics(url: string, onMetrics: (arg: KubeMetrics[]) => void) {
+export async function metrics(
+  url: string, onMetrics: (arg: KubeMetrics[]) => void, onError :(arg:Error) => void
```

Comment:
Add a space after the two `:`.

Reply:
You should also break it differently I think: leave the url + onMetrics on the previous line, and then add the `onError` arg in the next one, aligned to the opening `(`.